src/configs/getAndroidConfig.ts

```diff
@@ -40,15 +46,72 @@ export default function getAndroidConfig(theme: Theme): Platform {
         destination: `${_.upperFirst(_.camelCase(theme))}DesignTokens.kt`,
         className: _.upperFirst(_.camelCase(theme)) + "DesignTokens",
         packageName: "io.element.android.compound",
+        filter: function(token: TransformedToken) {
+          return !isFontType(token) && isCoreColor.matcher(token);
+        },
         options: {
           showFileHeader: false,
+          outputReferences: true,
           import: [
             "androidx.compose.ui.graphics.Color",
             "androidx.compose.ui.unit.*",
             "androidx.compose.ui.text.*",
           ],
         },
       },
+      // If we find a way to describe semantic colors, it might be possible to move this to 'common'
+      {
+        format: "compose/extra-colors",
+        destination: `CompoundColors.kt`,
+        className: "CompoundColors",
```

Comment:
Not entirely sure about conventions on Android, but do we need to prefix with `Compound` given that it's already indicated in the package path?

Reply:
We already have a `Colors.kt` file in a central module of the project, so if we don't set a different name we could have problems when using autocomplete.